nix/shell.nix

```diff
@@ -35,6 +35,7 @@ stdenv.mkDerivation {
   shellHook = ''
     mkdir -p ./Borg
     ln -sfn ${Borg}/include/borgms.h ./Borg/borgms.h
+    ln -sfn ${Borg}/* ./Borg/
```

Comment:
Just curious why this was needed? I thought the build worked without this.

Reply:
Yes, this was needed, but not for build.  This is to enable Bernardo to recompile within the container if necessary.

Reply:
OK - do we know the nature of the build error? I suppose I can test it out. I'm just curious because this seems to suggest two different ways of including and linking to Borg, and, I think, only one should be needed, but let's go ahead and keep it for now, and worry about perfection later.